www/install.rst

```diff
@@ -96,9 +96,9 @@ and OS-X systems that can be used to install the packages
 one by one. These installers are generally built to be compatible
 with the Python binaries available from python.org.
 
-*  `python <http://python.org/download/>`_
-*  `numpy <http://sourceforge.net/projects/numpy/files/NumPy/>`_
-*  `scipy <http://sourceforge.net/projects/scipy/files/scipy/>`_
+*  `python <https://www.python.org/downloads/>`_
+*  `numpy <https://sourceforge.net/projects/numpy/files/NumPy/>`_
+*  `scipy <https://sourceforge.net/projects/scipy/files/scipy/>`_
```

Comment:
Newer scipy releases are uploaded to Github Releases instead --- https://github.com/scipy/scipy/releases